docs/Behaviour - Server Side.md

```diff
@@ -26,7 +26,24 @@ A Sender MUST present the list of supported parameter constraints in `/supported
 - `urn:x-nmos:cap:format:sample_rate`
 - `urn:x-nmos:cap:format:sample_depth`
 
-## State of Sender
+## Status of Input
+
+An Input has the following states:
+- `no_signal` when there is no incoming signal to this Input.
+- `awaiting_signal` when the incoming signal is not stable. This is a transitional state until one of the other ones can be established.
+- `signal_present` when the incoming signal is present and stable.
+
+Debug information expressed through `debug` property SHOULD assist `no_signal` state and MAY assist the other states.
+
+## Status of Output
+
+An Output has the following states that describe the signal provided by associated Receiver(s) to this Output:
```

Comment:
Should this be a little more like the explanation of `no_essence` at the Sender?

Something like:

An Output has the following states that describe the signal at this Output:
- `no_signal` when there is no signal, which, if the Output has associated Receivers, can mean that one or more of these Receivers do not have a compliant active stream.

Reply:
Thank you! I added it.